release_build_files/readme.md

```diff
@@ -570,7 +570,7 @@ code.
 ### 8.4.0
 -   Changes
     -   Firestore: Added `operator==` and `operator!=` for `SnapshotMetadata`,
-        `Settings`, and `QuerySnapshot`.
+        `Settings`, and `QuerySnapshot`, `SetOptions`, and `DocumentChange`.
```

Comment:
I guess this will have a merge conflict with my other PR, but that's ok. I'll rebase when needed :)

Reply:
Nit: extra `and`.